lib/form/html.js

```diff
@@ -137,8 +141,31 @@ const renderer = (type, errors) => {
         render += `<div id="${input.id}" class="form--field ${required}">`;
       }
 
-      // Add error messaging and required
-      render += addRequired(addError(html, input, errors), input);
+      // Wraps repeatable attributes around div
+      if (Array.isArray(input.inputs)) {
+        context.forEach((data, index) => {
+          render += `<div id="sub-${input.id}" class="form-repeatable">`;
+          html = nunjucks.renderString(input.html, data);
+
+          // Add error messaging and required
+          render += addRequired(addError(html, input, errors), input);
+          render += `<button type="button" class="delete-button" id="${input.id}-del--${index}">Delete</button>`;
+          render += '</div>';
+        });
+        render += `<button type="button" class="add-button" id="${input.id}-add">Add</button>`;
+      }
+      else if (input.repeatable && input.repeatable === true) {
+        render += `<div id="sub-${input.id}" class="form-repeatable">`;
+
+        // Add error messaging and required
+        render += addRequired(addError(html, input, errors), input);
+        render += '</div>';
+        render += `<button type="button" class="add-button" id="${input.id}-add">Add</button>`;
```

Comment:
Why are you rendering the button outside of the containing div? Wouldn't we want this wrapped to allow us to style them together?